js/base/Exchange.js

```diff
@@ -2682,7 +2682,7 @@ module.exports = class Exchange {
         return undefined;
     }
 
-    parseAccount (account) {
```

Comment:
What's the reasoning behind this renaming?

Reply:
`parseAccount` is already a method that serves a different purpose, it's written on a number of exchanges

Reply:
@samgermain I see but I don't find `typeToAccount` a very clear name, @kroitor are you ok with this name?